Makefile

```diff
@@ -293,22 +276,29 @@ $(BUILT)/standalone/content/l10n/%/loop.properties: locale/%/standalone.properti
 
 # add-on
 add_on_jsx_files=$(wildcard add-on/panels/js/*.jsx)
+add_on_js_files=$(wildcard add-on/panels/js/*.js)
+add_on_test_files=$(wildcard add-on/panels/test/*.js)
+add_on_module_jsm_files=$(wildcard add-on/chrome/modules/*.jsm)
+add_on_module_js_files=$(wildcard add-on/chrome/modules/*.js)
 built_add_on_js_files=$(patsubst add-on/panels/js/%.jsx, \
 	 $(BUILT)/add-on/chrome/content/panels/js/%.js, \
-	 $(add_on_jsx_files))
-
-$(BUILT)/add-on/chrome/content/panels/js/%.js: add-on/panels/js/%.jsx
-	@mkdir -p $(@D)
-	$(BABEL) $< --out-file $@
+	 $(add_on_jsx_files)) $(patsubst add-on/panels/js/%.js, \
+ 	 $(BUILT)/add-on/chrome/content/panels/js/%.js, \
+ 	 $(add_on_js_files)) $(patsubst add-on/panels/test/%.js, \
+ 	 $(BUILT)/add-on/chrome/content/panels/test/%.js, \
+ 	 $(add_on_test_files)) $(patsubst add-on/chrome/modules/%.jsm, \
+ 	 $(BUILT)/add-on/chrome/content/modules/%.jsm, \
+ 	 $(add_on_module_jsm_files)) $(patsubst add-on/chrome/modules/%.js, \
+ 	 $(BUILT)/add-on/chrome/content/modules/%.js, \
+ 	 $(add_on_module_js_files))
 
 add_on_vendor_jsx_files=$(wildcard add-on/panels/vendor/*.jsx)
+add_on_vendor_js_files=$(wildcard add-on/panels/vendor/*.js)
 built_add_on_vendor_js_files=$(patsubst add-on/panels/vendor/%.jsx, \
 	 $(BUILT)/add-on/chrome/content/panels/vendor/%.js, \
-	 $(add_on_vendor_jsx_files))
-
-$(BUILT)/add-on/chrome/content/panels/vendor/%.js: add-on/panels/vendor/%.jsx
-	@mkdir -p $(@D)
-	$(BABEL) $< --out-file $@
+	 $(add_on_vendor_jsx_files)) $(patsubst add-on/panels/vendor/%.js, \
+	 	 $(BUILT)/add-on/chrome/content/panels/vendor/%.js, \
+	 	 $(add_on_vendor_js_files))
```

Comment:
Indention looks wrong here, but it'll be cleaned up when having `$(patsubst) on own lines anyway.